inspirehep/modules/forms/static/js/forms/conferences_typeahead.js

```diff
@@ -50,8 +50,9 @@ define([
           url: '/api/conferences?q=conferenceautocomplete:%QUERY*',
           filter: function(response) {
             return $.map(response.hits.hits, function(el) {
-              el.metadata.city = el.metadata.address[0].cities[0];
-              el.metadata.country = el.metadata.address[0].country_code;
+              obj = el.metadata.address.find(function (obj) { return obj.hasOwnProperty('cities') });
```

Comment:
I'm using this instead of simply adding an `if(el.metadata.address[0].cities)` because in some cases (example, [this](https://labs.inspirehep.net/api/conferences/1646000) one) the object containing `cities` key is not the first element in the `address` array.